src/next-transpile-modules.js

```diff
@@ -114,29 +113,17 @@ const withTmInitializer = (transpileModules = []) => {
           );
 
           if (nextCssLoader) {
-            if (isModernIssuer(nextCssLoader.issuer)) {
-              nextCssLoader.issuer.or = nextCssLoader.issuer.or ? nextCssLoader.issuer.or.concat(includes) : includes;
-              nextCssLoader.issuer.not = excludes;
-            } else {
-              nextCssLoader.issuer.include = nextCssLoader.issuer.include
-                ? nextCssLoader.issuer.include.concat(includes)
-                : includes;
-              nextCssLoader.issuer.exclude = excludes;
-            }
+            nextCssLoader.issuer.or = nextCssLoader.issuer.and ? nextCssLoader.issuer.and.concat(includes) : includes;
```

Comment:
is this supposed to `concat` to `issuer.or`?

you're deleting the `and` property right after, so I'm assuming this doesn't do anything in the `then` case.

Reply:
Well, we want to use `or` so it matches the app directory, or one of the directory in the node_modules (the ones supplied as parameters of the plugin) 🤔

I removed the `delete` just in case.

Reply:
oh I read this wrong. of course, `concat` is immutable.